requirements.txt

```diff
@@ -1,8 +0,0 @@
-cachetools>=5.3.0
```

Comment:
If we remove the `requirements.txt` and `requirements_test.txt`, then we need to update the Installation part in README [and ](https://github.com/understandable-machine-intelligence-lab/Quantus#installation) and in readthedocs https://quantus.readthedocs.io/en/latest/getting_started/installation.html which both propose this type of installation?

Reply:
I didn't notice at first, thanks for pointing out! So, I have found the following section in the readme:
```
Alternatively, you can simply install Quantus with [requirements.txt](https://github.com/understandable-machine-intelligence-lab/Quantus/blob/main/requirements.txt).
Note that this installation requires that either [PyTorch](https://pytorch.org/) or [TensorFlow](https://www.TensorFlow.org) are already installed on your machine.
```setup
pip install -r requirements.txt
```
```
And this is not true. Running `pip install -r requirements.txt` will only install dependencies. 
I have described the installation from source in `CONTRIBUTING.md` on lines 45-55.

To sum up: I will remove outdated sections from the readme and readthedocs title page.

Reply:
Great, thank you.